compiler/main/driver.cpp

```diff
@@ -1200,9 +1200,9 @@ static ArgumentDescription arg_desc[] = {
  {"warn-tuple-iteration", ' ', NULL, "Enable [disable] warnings for tuple iteration", "n", &fNoWarnTupleIteration, "CHPL_WARN_TUPLE_ITERATION", setWarnTupleIteration},
  {"warn-special", ' ', NULL, "Enable [disable] special warnings", "n", &fNoWarnSpecial, "CHPL_WARN_SPECIAL", setWarnSpecial},
 
- {"compiler-library-parser", ' ', NULL, "Enable [disable] using compiler library parser", "N", &fCompilerLibraryParser, "CHPL_COMPILER_LIBRARY_PARSER", NULL},
- {"debug-trace", ' ', NULL, "Enable [disable] debug-trace output when using compiler library parser", "N", &fDebugTrace, "CHPL_DEBUG_TRACE", NULL},
- {"break-on-query-hash", ' ' , NULL, "Break when query with given hash value is executed when using compiler library parser", "U", &fBreakOnQueryHash, "CHPL_BREAK_ON_QUERY_HASH", NULL},
+ {"dyno", ' ', NULL, "Enable [disable] using dyno compiler library", "N", &fDynoCompilerLibrary, "CHPL_DYNO_COMPILER_LIBRARY", NULL},
+ {"dyno-debug-trace", ' ', NULL, "Enable [disable] debug-trace output when using dyno compiler library", "N", &fDynoDebugTrace, "CHPL_DYNO_DEBUG_TRACE", NULL},
+ {"dyno-break-on-query-hash", ' ' , NULL, "Break when query with given hash value is executed when using dyno compiler library", "U", &fDynoBreakOnQueryHash, "CHPL_DYNO_BREAK_ON_QUERY_HASH", NULL},
```

Comment:
I would like to rename this to `--dyno-break-query-hash` because the current name is a bit too long for my tastes. Alternatives are `--break-on-query` or `--break-on-hash`, but both of those might come with some measure of confusion (the first because developers might expect to input a query name, the second because "hash for what?").

Removing `-on-` makes the length more acceptable to me.

Reply:
I think we should call it `--dyno-break-on-hash`

Reply:
I'm fine with any of these names, whether renamed or not.

Reply:
>  the current name is a bit too long for my tastes.

> I think we should call it `--dyno-break-on-hash`

> I'm fine with any of these names, whether renamed or not.

`--dyno-break-on-hash` it is 😄